invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RecordVersionsList.js

```diff
@@ -36,6 +37,11 @@ const RecordVersionItem = ({ item, activeVersion }) => {
             {item.publication_date}
           </p>
         </Grid.Column>
+        <Grid.Column width={12}>
```

Comment:
In semantic-ui, columns are max `16`. I suspect you wanted to go to a new row here.
In that case, you have to:
```
</Grid.Row>
<Grid.Row columns={1}>
...
</Grid.Row>

Reply:
Thank you. I changed it in the other commit